app/controllers/dashboard/districts_controller.rb

```diff
@@ -19,7 +17,7 @@ def show
     else
       Date.current.advance(months: -1)
     end
-    @data = DistrictReportService.new(facilities: @district.facilities, selected_date: @selected_date).call
+    @data = DistrictReportService.new(district: @district, selected_date: @selected_date).call
```

Comment:
I actually wanted to allow either, but for this iteration it needs to take a district because we use 

```
district.patients.with_hypertension
``` 

to get registration counts. I imagine we could find a way to go from the set of facilities to get registration counts, but it would be more indirect I think.